clients/command-kafka/src/main/java/org/eclipse/hono/client/command/kafka/KafkaBasedCommandResponseSender.java

```diff
@@ -60,6 +60,9 @@ public KafkaBasedCommandResponseSender(
 
         final HonoTopic topic = new HonoTopic(HonoTopic.Type.COMMAND_RESPONSE, response.getTenantId());
         log.trace("publish command response [{}]", response);
+        if (response.getMessagingType() != getMessagingType()) {
+            // TODO log to span
```

Comment:
did you leave this out intentionally?

Reply:
Yes. The span is currently created in the parent class as part of the following `sendAndWaitForOutcome` invocation. I want to change that in a followup commit, creating the span here already, so that also a specific span name can be used.

Reply:
ok